lib/mongodb.js

```diff
@@ -0,0 +1,46 @@
+import { MongoClient } from 'mongodb'
```

Comment:
Could you add a URL at the top of this file in a comment for where this code is from?

```
// code taken from: https://dogland.com

import { MongoClient } ...
```

Reply:
i can't find this file at that URL. All I can find is this: https://github.com/vercel/next.js/blob/canary/examples/with-mongodb/lib/mongodb.js